### DISCUSSION TITLE
conf.d permissions on Fedora

### DISCUSSION THREAD

**Discussion by mikwee** [nginx/nginx#414] [Q&A]
I'm using NGiNX on Fedora, I put a config file in `conf.d/`, but I have no idea what its permissions should be, and to which user it should belong. No matter what I try, I get this error:
`nginx: [emerg] open() "/etc/nginx/conf.d/vichan.conf" failed (13: Permission denied) in /etc/nginx/nginx.conf:35`

If you could help, that would be awesome. Thanks!

**Comment by Maryna-f5** [comment]
This discussion has been inactive for more than 90 days, closing it.
